.travis.yml

```diff
@@ -0,0 +1,60 @@
+version: ~> 1.0
+language: generic
+group: travis_latest
+
+# Only build for develop
+branches:
+  only:
+    - develop
+
+matrix:
+  fast_finish: true
+  include:
+    - python: 3.8
+      name: "Power PC"
+      os: linux
+      arch: ppc64le
+      if: type = cron
```

Comment:
I'll be honest, I'm not 100% sure this will work. I have enabled the cron job on `develop` to be daily for now (eventually we can change it to weekly). So maybe it'll get triggered tomorrow, but it's not clear to me that the cron job will run if this doesn't get merged first.